weblate/templates/keyboard-shortcuts.html

```diff
@@ -0,0 +1,166 @@
+              </td>
+              <td>{% trans "Focus comment editor."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Ctrl</kbd> + <kbd>M</kbd> {% trans "or" %} <br>
+                <kbd>Cmd</kbd> + <kbd>M</kbd>
+              </td>
+              <td>{% trans "Shows Automatic suggestions tab."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Ctrl</kbd> + <kbd>1</kbd> {% trans "to" %}
+                <kbd>Ctrl</kbd> + <kbd>9</kbd> {% trans "or" %} <br>
+                <kbd>Cmd</kbd> + <kbd>1</kbd> {% trans "to" %}
+                <kbd>Cmd</kbd> + <kbd>9</kbd>
+              </td>
+              <td>{% trans "Copies placeable of given number from source string."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Ctrl</kbd> + <kbd>M</kbd> {% trans "followed by" %}
+                <kbd>1</kbd> {% trans "to" %} <kbd>9</kbd> {% trans "or" %} <br>
+                <kbd>Cmd</kbd> + <kbd>M</kbd> {% trans "followed by" %}
+                <kbd>1</kbd> {% trans "to" %}
+                <kbd>9</kbd>
+              </td>
+              <td>{% trans "Copy the machine translation of given number to current translation."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Ctrl</kbd> + <kbd>I</kbd> {% trans "followed by" %}
+                <kbd>1</kbd> {% trans "to" %}
+                <kbd>9</kbd> {% trans "or" %} <br>
+                <kbd>Cmd</kbd> + <kbd>I</kbd> {% trans "followed by" %}
+                <kbd>1</kbd> {% trans "to" %}
+                <kbd>9</kbd>
+              </td>
+              <td>{% trans "Ignore one item in the list of failing checks."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Ctrl</kbd> + <kbd>J</kbd> {% trans "or" %}
+                <br> <kbd>Cmd</kbd> + <kbd>J</kbd>
+              </td>
+              <td>{% trans "Shows the Nearby strings tab."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Ctrl</kbd> + <kbd>S</kbd> {% trans "or" %}
+                <br> <kbd>Cmd</kbd> + <kbd>S</kbd>
+              </td>
+              <td>{% trans "Focus search field."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Ctrl</kbd> + <kbd>O</kbd> {% trans "or" %} <br>
+                <kbd>Cmd</kbd> + <kbd>O</kbd>
+              </td>
+              <td>{% trans "Copy source string."%}</td>
+            </tr>
+            <tr>
+              <td><kbd>Ctrl</kbd> + <kbd>Y</kbd> {% trans "or" %}
+                <br> <kbd>Cmd</kbd> + <kbd>Y</kbd>
+              </td>
+              <td>{% trans "Toggle the Needs editing checkbox."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>→</kbd> and <br> <kbd>←</kbd>
+              </td>
+              <td>{% trans "Browse the next translation strings. <br> Browse the previous translation strings."%}</td>
```

Comment:
Let's split the string here:

```suggestion
              <td>
                {% trans "Browse the next translation strings." %}
                <br>
                {% trans "Browse the previous translation strings."%}
              </td>
```